try-it.md

```diff
@@ -435,7 +359,7 @@ awk '{print $1}' > \
 /opt/metal3-dev-env/ironic/html/images/centos-updated.qcow2.md5sum
 ```
 
-### 2.5. Directly Provisioning Bare Metal Hosts
+### 2.5. Directly Provision BareMetalHost
```

Comment:
I think the same style guide rule would apply here, too.

Reply:
Fixed.